feature-libs/order/components/order-history/order-history.component.html

```diff
@@ -53,92 +53,242 @@ <h2 *ngIf="!type.replenishmentOrder">
-            <tbody>
-              <tr
-                *ngFor="let order of type.orderHistory.orders"
-                (click)="goToOrderDetail(order)"
-              >
-                <td class="cx-order-history-code">
-                  <div class="cx-order-history-label">
-                    {{ 'orderHistory.orderId' | cxTranslate }}
-                  </div>
-                  <a
-                    [routerLink]="
-                      {
-                        cxRoute: 'orderDetails',
-                        params: order
-                      } | cxUrl
-                    "
-                    class="cx-order-history-value"
-                  >
-                    {{ order?.code }}</a
-                  >
-                </td>
-                <td class="cx-order-history-placed">
-                  <div class="cx-order-history-label">
-                    {{ 'orderHistory.date' | cxTranslate }}
-                  </div>
-                  <a
-                    [routerLink]="
-                      {
-                        cxRoute: 'orderDetails',
-                        params: order
-                      } | cxUrl
-                    "
-                    class="cx-order-history-value"
-                    >{{ order?.placed | cxDate: 'longDate' }}</a
-                  >
-                </td>
-                <td class="cx-order-history-status">
-                  <div class="cx-order-history-label">
-                    {{ 'orderHistory.status' | cxTranslate }}
-                  </div>
-                  <a
-                    [routerLink]="
-                      {
-                        cxRoute: 'orderDetails',
-                        params: order
-                      } | cxUrl
-                    "
-                    class="cx-order-history-value"
-                  >
-                    {{
-                      'orderDetails.statusDisplay_' + order?.statusDisplay
-                        | cxTranslate
-                    }}</a
-                  >
-                </td>
-                <td class="cx-order-history-total">
-                  <div class="cx-order-history-label">
-                    {{ 'orderHistory.total' | cxTranslate }}
-                  </div>
-                  <a
-                    [routerLink]="
-                      {
-                        cxRoute: 'orderDetails',
-                        params: order
-                      } | cxUrl
-                    "
-                    class="cx-order-history-value"
-                  >
-                    {{ order?.total.formattedValue }}</a
-                  >
-                </td>
-              </tr>
-            </tbody>
-          </table>
+
+          <ng-container *cxFeatureLevel="'5.1'">
+            <table
+              role="table"
+              id="order-history-table"
+              class="table cx-order-history-table"
+              [ngClass]="{ 'cx-order-history-table-po': hasPONumber }"
```

Comment:
Could we move the featureLevel condition into the class on `hasPONumber` so that we can add deprecation notes to the newly exposed flag? We shouldn't need to duplicate the table in the template if the flag itself depends on the feature level.

Reply:
done thanks